lib/components/viewers/RouteRow.js

```diff
@@ -59,41 +63,30 @@ export const RouteRowButton = styled(Button)`
   }
 `
 
-export const RouteRowElement = styled.span`
-  padding-right: 8px;
-`
 export const OperatorImg = styled.img`
   height: 25px;
-  margin-right: 8px;
 `
 
 export const ModeIconElement = styled.span`
+  display: inherit; // Allows for equal sizing and vertical centering of mode icons
   height: 28px;
-  padding-left: inherit;
   width: 28px;
 `
 
 const RouteNameElement = styled.span`
   flex-shrink: 0;
   font-size: 16px;
   font-weight: 400;
-  margin-left: 8px;
 `
 const RouteDetailsContainer = styled.div`
-  align-items: center;
-  display: grid;
-  grid-template-columns: repeat(4, auto);
+  ${centeredStyle}
+  gap: 12px;
   justify-items: center;
+  margin-left: 5px;
 `
 const RouteLongNameElement = styled.span`
   font-size: 16px;
   font-weight: 500;
-  /*
-    Top margin is same as border-top-width+margin-top of the default route renderer,
-    so that the baseline of the route short name aligns with that
-    of the route long name.
-  */
-  margin: 2px 0 0 15px;
```

Comment:
This feels scary to remove -- how is this behavior carried over!

Reply:
Our goal was to remove hard coded margins and replace it with gaps. This styling didn't really help anywhere (to our knowledge)

Reply:
Just to be safe I think we should assign @binh-dam-ibigroup as a final reviewer, who I think put in this line

Reply:
Sure, @binh-dam-ibigroup let us know your thoughts on this!

The margin was removed because when all the elements are centered, this line dramatically throws off the spacing. Particularly the margin-left. You can see the screenshots here: https://github.com/opentripplanner/otp-react-redux/pull/854#issuecomment-1513772351

Reply:
The left margin on the route long name is not really needed. The top margin was to align the baseline (the bottoms of characters such as "3", "a" - but that depends on the font) of the route number and route long name, which can be nice to have.
The right margin leaves too much space, however, with the introduction of the View Route Details button.